internal/logrus_test.go

```diff
@@ -19,7 +20,7 @@ func TestLogrusLoggerLevel(t *testing.T) {
 
 	l.Error("test error")
 
-	assert.Contains(t, buffer.String(), "\"level\":\"error\"")
+	assert.Contains(t, buffer.String(), "\"level\":2")
```

Comment:
@jmdalmeida Not sure why this test was failing, but I had to change it to the numerical value to be able to pass.
Can you validate this please?

Reply:
Oops... my bad. When I applied our [forked version of logrus](https://github.com/sirupsen/logrus/pull/698) (to be compliant with the Syslog Protocol, that Graylog now enforces), I must've forgotten to run the tests. This is expected.

Reply:
I got you! :)